src/language/CodeInspection.js

```diff
@@ -608,11 +608,19 @@ define(function (require, exports, module) {
             toggleCollapsed(prefs.get(PREF_COLLAPSED), true);
         });
     
-    prefs.definePreference(PREF_ASYNC_TIMEOUT, "number", 10000);
+    prefs.definePreference(PREF_ASYNC_TIMEOUT, "number", 10000, {
+        description: Strings.DESCRIPTION_ASYNC_TIMEOUT
+    });
 
-    prefs.definePreference(PREF_PREFER_PROVIDERS, "array", []);
+    prefs.definePreference(PREF_PREFER_PROVIDERS, "array", [], {
+        description: Strings.DESCRIPTION_LINTING_PREFER,
+        values: ["JSLint", "JSHint"],
```

Comment:
Yes, it could be done, I guess. My point, though, it should not have default in the property definition (there's no default for this pref).